dev-docs/feature-conditional.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: page
+title: Conditional Ad Units - <font color="red">NEW!</font>
+description: Using labels for conditional bids
```

Comment:
This feature is referred to alternately as "conditional ad units" and "conditional bids".  I think we should standardize on one.

My vote is for "conditional ad unit" since the logic is captured inside the ad unit, and Prebid in general already calls a lot of things "bids" and makes it difficult to draw a clear distinction between client side "bids" and the actual "bids" returned by the server.